deft.js

```diff
@@ -115,29 +115,17 @@ deft.normalizeFilePaths = function(files) {
  * @return {string}
  *
  * @examples
- * deft.getUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
- * // => 'https://raw.githubusercontent.com/lodash/lodash/master/lodash.js'
+ * deft.getInformationUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
+ * // => 'https://api.github.com/repos/lodash/lodash/contents/lodash.js?ref=master'
  *
- * deft.getUrl('underscore.js', ['jashkenas/underscore', '1.6.0', 'underscore.js']);
```

Comment:
That's not true :( The logic for downloading and extracting archives is separate from downloading files. I just tried with http://code.jquery.com/jquery-1.10.1.js and it didn't work when I removed the code you've removed here.